src-docs/src/routes.js

```diff
@@ -353,6 +355,7 @@ const navigation = [
     items: [
       BreadcrumbsExample,
       ButtonExample,
+      SplitButtonExample,
```

Comment:
I'm curious why under Forms?  It's not an input... although it takes some look-feel from SuperSelect.

Reply:
It's under navigation because that's where `button` is referenced. Mental model wise, this is a different variant of button and should be together with all the other buttons, including button group, icon button, etc.

Reply:
I would argue that it belongs in the button page, after button group, before ghost.

Reply:
I feel like the SplitButton is related to Button in a similar way that SuperSelect is related to ComboBox.
The API and functionality of SlitButton is a far-enough deviation from Button that I'd think it might be confusing to have it as a "mode" of Button (on the same doc page).
Your thoughts?